.gitignore

```diff
@@ -2,3 +2,4 @@
 composer.lock
 /phpunit.xml
 .phpunit.result.cache
+.idea
```

Comment:
To be inline with `laravel/framework` .gitignore file, I would suggest you to add a forward slash in front of this folder name

`/.idea`

Reference: https://github.com/laravel/framework/blame/7.x/.gitignore

Reply:
I suggest this should be handled in your global gitignore rather than adding it here